pages/common/svn.md

```diff
@@ -18,6 +18,6 @@
 
 `svn ci -m {{commit log message}} {{[PATH...]}}`
 
-- Show detailed help:
```

Comment:
We can keep both. And 10 should be in tokens (`{{}}`)

Reply:
Should be fixed now.